### PR TITLE
TST: port seqview tests

### PR DESCRIPTION
**Pull request by fredjaya**
no new xfails, main changes so tests work with the new implementation are:
- adding alphabets arg to SeqView
- replacing value with str_value

nearly there...

### CODE REVIEW

File: tests/test_core/test_sequence.py [hunk 1]
```diff
@@ -1275,504 +1275,12 @@ def test_get_translation_trim_stop():  # ported
     assert str(aa) == "I*L"
 
 
-@pytest.mark.parametrize("start", (None, 0, 1, 10, -1, -10))
```
Comment (KatherineCaley) [inline tests/test_core/test_sequence.py:1278]: I think we should be leaving the old tests in place

**Review by GavinHuttley — CHANGES_REQUESTED** [review]
leave old tests in place, just comment as # ported

**Comment by fredjaya** [comment]
Will do. Should all the ones deleted up until now be re-added?

**Comment by fredjaya** [comment]
Ported a few more tests and marked those in place. Let me know if I should re-add all the previous ones deleted before this PR

**Review by GavinHuttley — CHANGES_REQUESTED** [review]
All tests of code that remains in the original sequence, moltype, alignment and genetic_code modules, should also remain.

**Review by GavinHuttley — APPROVED** [review]
awesome!

**Pull request merged**